billing/Earthfile

```diff
@@ -0,0 +1,12 @@
+VERSION 0.6
+
+FROM alpine/git:2.36.3
+
+# Estimates monthly active users of a GIT remote repo, based on past commits
+active-users:
+  LOCALLY
+  ARG repoUrl
+  RUN test $repoUrl
+  RUN git clone --bare --filter=blob:none $repoUrl repo;
+  RUN cd repo; \
+      git --no-pager log -s --format="%ae" --since=30.days | grep -v @users.noreply.github.com | sort | uniq -c | awk '$1 >= 3 {print $0}' | wc -l
```

Comment:
Oh didn't realize that in our query. Actually we should exclude both:
>Note: If you created your account on GitHub.com after July 18, 2017, your noreply email address for GitHub is an ID number and your username in the form of [ID+USERNAME@users.noreply.github.com](mailto:ID+USERNAME@users.noreply.github.com). If you created your account on GitHub.com prior to July 18, 2017, your noreply email address from GitHub is [USERNAME@users.noreply.github.com](mailto:USERNAME@users.noreply.github.com). You can get an ID-based noreply email address for GitHub by selecting (or deselecting and reselecting) Keep my email address private in your email settings.

https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/setting-your-commit-email-address#about-commit-email-addresses

Reply:
Ah I see - I personally use `USERNAME@users.noreply.github.com` in my gitconfig actually, to avoid exposing my email online. I was hoping we would cover that case as I've seen people do that commonly.

Anyway, perhaps we filter out both cases for now as you point out, since we have no good way to distinguish uses in gitconfig vs uses from the web.